pipeline/stop_places/sql/stop_places.sql

```diff
@@ -101,20 +101,96 @@ LANGUAGE SQL IMMUTABLE STRICT;
+  END IF;
+  RETURN NULL;
+END
+$$
+LANGUAGE plpgsql IMMUTABLE STRICT;
+
+
+/*
+ * Create a single key value pair element where the value is the given value if it is not empty
+ * Else returns null
+ */
+CREATE OR REPLACE FUNCTION delfi_attribute_get_value_xml(delfiid text, val text) RETURNS xml AS
+$$
+SELECT CASE
+  WHEN $2 IS NOT NULL THEN create_KeyValue($1, $2)
+END
+$$
+LANGUAGE SQL IMMUTABLE STRICT;
+
+
+/*
+ * Create a keyList element for stop places based on a delfi attribut to osm matching
+ * Optionally additional key value pairs can be passed to the function
+ * Returns null when no tag matching exists
+ */
+CREATE OR REPLACE FUNCTION ex_keyList_StopPlace(tags jsonb, additionalPairs xml DEFAULT NULL) RETURNS xml AS
+$$
+DECLARE
+  result xml;
+BEGIN
+  result := xmlconcat(
+    additionalPairs
+  );
+
+  IF result IS NOT NULL THEN
+    RETURN xmlelement(name "keyList", result);
+  END IF;
+
+  RETURN NULL;
+END
+$$
+LANGUAGE plpgsql IMMUTABLE;
+
+
+/*
+ * Create a keyList element for quays based on a delfi attribut to osm matching
  * Optionally additional key value pairs can be passed to the function
  * Returns null when no tag matching exists
  */
-CREATE OR REPLACE FUNCTION ex_keyList(tags jsonb, additionalPairs xml DEFAULT NULL) RETURNS xml AS
+CREATE OR REPLACE FUNCTION ex_keyList_Quay(tags jsonb, additionalPairs xml DEFAULT NULL) RETURNS xml AS
 $$
 DECLARE
   result xml;
 BEGIN
   result := xmlconcat(
     additionalPairs,
-    delfi_attribute_on_yes_xml('1120', tags->>'bench'),
-    delfi_attribute_on_yes_xml('1140', tags->>'passenger_information_display'),
-    delfi_attribute_on_yes_xml('1141', tags->>'passenger_information_display:speech_output')
+    delfi_attribute_on_yes_xml('1120', tags->>'bench'), -- waiting area with seat
+    delfi_attribute_on_yes_xml('1140', tags->>'passenger_information_display'), -- dynamic visual passenger information display
+    delfi_attribute_on_yes_xml('1141', tags->>'passenger_information_display:speech_output'),  -- with acoustic output
+    delfi_attribute_on_yes_xml('1150', tags->>'announcement'), -- automatic announcements
+    delfi_attribute_on_yes_xml('1170', tags->>'height'),  -- curb/platform height
+    delfi_attribute_on_yes_xml('1180', tags->>'width')   -- curb/platform width
+    --delfi_attribute_on_yes_xml('1190', tags->>''), -- distance between platform edge and center of track
+    --delfi_attribute_on_yes_xml('1200', tags->>''), -- high curb with track guidance
+    --delfi_attribute_on_yes_xml('1201', tags->>''), -- high curb with track guidance and double cove
+    --delfi_attribute_on_yes_xml('1202', tags->>''), -- high curb without track guidance
+    --delfi_attribute_on_yes_xml('1203', tags->>''), -- 'combiboard' with track guidance
+    --delfi_attribute_on_yes_xml('2071', tags->>''), -- tactile/visual floor indicators in the entrance area with locating strips
+    --delfi_attribute_on_yes_xml('2140', tags->>''), -- entry in the middle of the road
```

Comment:
Can you please format them like so for every "ex_keyList" function:

```SQL
-- 1120: waiting area with seat
delfi_attribute_on_yes_xml('1120', tags->>'bench'),
```
(comment on top and DELFI id at the start)

Just because this visually separates them better which becomes more important later when extracting an DELFI attribute is not just a matter of one line.